tests/utils/SparseGridTest.cpp

```diff
@@ -217,13 +256,14 @@ TEST_F(GetNearestTest, Equal)
     }
 
     typename SparsePointGridInclusive<Point>::Elem result;
-    //The actual call to test.
+    // The actual call to test.
     const bool success = grid.getNearest(target, grid_size, result, SparsePointGridInclusive<Point>::no_precondition);
 
     ASSERT_TRUE(success);
-    ASSERT_TRUE(result.val == expected1 || result.val == expected2) << "getNearest reported the nearest point to be " << result.val << " (distance " << vSize(target - result.val) <<
-            "), but it should've been " << expected1 << "(distance " << vSize(expected1 - target) <<
-            ") or " << expected2 << " (distance " << vSize(expected2 - target) << ").";
+    ASSERT_TRUE(result.val == expected1 || result.val == expected2)
+      << "getNearest reported the nearest point to be " << result.val << " (distance " << vSize(target - result.val)
+      << "), but it should've been " << expected1 << "(distance " << vSize(expected1 - target) << ") or " << expected2 << " (distance "
+      << vSize(expected2 - target) << ").";
```

Comment:
Indent with 4 spaces.

This line can definitely be broken up in a more readable way. Seems rather mechanical.

Reply:
I would suggest to add a FIXME: once we have fmt as dependency. This will be automatically added once we added spdlog. The line can then be changed to:
```
    ASSERT_TRUE(result.val == expected1 || result.val == expected2) << fmt::format(
    "getNearest reported the nearest point to be {} (distance {}), but it should've been {} (distance {}) or (distance {})", 
     result.val, vSize(target - result.val), expected1, vSize(expected1 - target),  expected2,  vSize(expected2 - target));
```